src/js/blue-button/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+
+import form from './form';
+
+export default combineReducers({
+  form,
```

Comment:
@austin-martinez do you have a preference for these? I usually leave them so that there is less of a chance for errors, and it's easier to add/remove lines.